browser/ui/views/tabs/features.cc

```diff
@@ -30,6 +30,14 @@ BASE_FEATURE(kBraveVerticalTabs,
              "BraveVerticalTabs",
              base::FEATURE_DISABLED_BY_DEFAULT);
 
+BASE_FEATURE(kBraveChangeActiveTabOnScrollEvent,
+             "BraveChangeActiveTabOnScrollEvent",
+#if BUILDFLAG(IS_LINUX)
+             base::FEATURE_ENABLED_BY_DEFAULT);
+#else
+             base::FEATURE_DISABLED_BY_DEFAULT);
```

Comment:
shouldn't it be just enabled for any OS?

Reply:
I'd like to have it mirror browser_defaults::kScrollEventChangesTab